languages.toml

```diff
@@ -3056,3 +3057,17 @@ language-servers = [ "templ" ]
 [[grammar]]
 name = "templ"
 source = { git = "https://github.com/vrischmann/tree-sitter-templ", rev = "ea56ac0655243490a4929a988f4eaa91dfccc995" }
+
+[[language]]
+name = "handlebars"
+injection-regex = "hbs"
+roots = ["package.json"]
+file-types = ["hbs"]
```

Comment:
`.hbs` is just the handlebars extension, no? Templating languages are tricky to recognize because they can template any other language. AFAIK handlebars isn't HTML-specific so using an HTML-specific parser for all `.hbs` will give the wrong highlights for some files

Reply:
Yes `.hbs` is just handlebars, this grammar is made for `ember.js`'s adaptation of handlebars (glimmer) templates which has some additional things e.g. components (tags starting with an uppercase letter), `yield`. html highlighting etc.